test2.py

```diff
@@ -0,0 +1,10 @@
+import pandas as pd 
```

Comment:
testing out your function after moving it to utils.py (you can keep more than one function in there unlike matlab)